src/CattyTests/Scene/SceneTests.swift

```diff
@@ -58,58 +58,58 @@ final class SceneTests: XCTestCase {
     }
 
     func testTouchConversionCenter() {
-        let scaledScene = SceneBuilder(project: ProjectMock(width: self.screenSize.width * 2, andHeight: self.screenSize.height * 2)).build()
+        let scaledScene = StageBuilder(project: ProjectMock(width: self.screenSize.width * 2, andHeight: self.screenSize.height * 2)).build()
```

Comment:
please rename this variable and the following scaledScene/scene variables in the functions below.